ericsmodule/src/main/java/sdoward/com/ericsmodule/BeerView.java

```diff
@@ -5,4 +5,6 @@
 public interface BeerView {
     void showError(Throwable throwable);
     void displayBeers(List<Beer>beers);
+    void displayBeerCount(int beerCount);
```

Comment:
This should be `displayAverageBeerPercentage(double beerPercantage)`, beer percentage needs decimels so an int will not work